.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
```

Comment:
```suggestion
Name: Bug report
```

Reply:
I assume it should work but I'm not sure. lowercased name is referenced in github doc 
> To be included in the community profile checklist, issue templates must be located in the .github/ISSUE_TEMPLATE folder and contain valid name: and about: YAML front matter.
> from https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates

@wtam2018 Could you elaborate more why do you propose that changes?
P.S. Seems odo repo uses `name` and `about` as well https://raw.githubusercontent.com/openshift/odo/master/.github/ISSUE_TEMPLATE/Documentation.md

Reply:
@sleshchenko  Nevermind those suggestions.   Those tags in fact need to be all lowercases.